src/vs/editor/browser/controller/mouseHandler.ts

```diff
@@ -557,6 +617,7 @@ class MouseDownState {
 	private _lastMouseDownPositionEqualCount: number;
 	private _lastMouseDownCount: number;
 	private _lastSetMouseDownCountTime: number;
+	private _lastMouseDownEvent: 'mouseDown' | 'drag';
```

Comment:
pick a better name for this. Perhaps just a boolean `isDragAndDrop` would do. This boolean should be set only in the `start` method above